docs/TESTING.md

```diff
@@ -63,12 +49,24 @@ The tests are run against a live application, built in production mode. To make
 work consistently and do not depend on a specific schedule or realtime vehicle locations, we use
 [WireMock](http://wiremock.org/) to record and playback the V3 API responses.
 
-To run the tests, do the following:
+Prerequisites for running the tests:
+
+* Docker
+  * `brew cask install docker`
+  * Start Docker Desktop from the dock or Applications; this only has to be done
+    once, after which it will auto-start on login by default
+* Wiremock
+  * `brew cask install java --no-quarantine`
+    * This option is currently required on OS X 10.15+ due to Gatekeeper
+      changes. Ref: https://github.com/Homebrew/homebrew-cask/issues/70798
+  * `brew cask install wiremock-standalone`
+* Ensure `WIREMOCK_PATH` points to the Wiremock JAR file; with brew cask this
```

Comment:
Should this be listed as an envar or is this used inline with each run?

Reply:
Ah, do you mean the wording of the sentence doesn't make it clear that `WIREMOCK_PATH` is in fact an environment variable and not some other kind of thing?

Reply:
Yes, and it doesn't seem to be listed in the envars section(s).

Reply:
It is listed in `ENVIRONMENT.md` if that's what you mean? This doc itself (`TESTING`) doesn't have a section about env vars.

Reply:
FWIW my first guess on seeing an all-caps identifier in this context is that it would be an environment variable, but we could be clear while maintaining concision by saying "Ensure the environment variable `WIREMOCK_PATH`..."

Reply:
I did change the wording here 👍